### PR TITLE
Feature Section

### PR DESCRIPTION
**Pull request by jodavid444**
# Awesome Books Application

- [x] create the index.html file, add the form section
- [x] create a style.css file
- [x] create an app.js file and implement functionalities to make the web application active
- [x] update readme.MD file
- [x]  update app.js file modified error message
- [x]  update style.css file modify title font size

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Hi @jodavid444 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no linters running. Kindly fix that and resubmit your project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @jodavid444 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: 

You have submitted a project, that has failed linter checks. Please check and fix the CSS linter errors using this command `npx stylelint "**/*.{css,scss}" --fix` and request for another review once you are ready.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+button.addEventListener('click', (event)=> {
+event.preventDefault();
+    if(titleInput.value == "" && authorInput) {
+        return
+    }
+
+    else {
+        const booklist = document.querySelector('#book-list')
+        const newbook = document.createElement('div');
+        const newAuthor = document.createElement('div');
```
Comment (uwadonat) [inline app.js]: - [ ] Let's  us a function to add a new book to the collection, with title and author. 
`Hint`: For book collection you can use an array of objects.
- [ ] Also `localStorage` to preserve data in the browser's memory.

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi@Team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good pull request message ✔️
- Professional readme file ✔️
almoo




## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — APPROVED** [review]
Hi @jodavid444, 

# SATATUS: APPROVED 😃 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️
- Descriptive PR ✅
- Correct Github flow ☑️
- Professional README file 👍🏽
- The app works as expected 💯

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @jodavid444, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that is a pair programming and doesn't have commits from both partners in your group. Make sure to create a new branch and work with your pair programming partner.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by jodavid444** [comment]
@sja-thedude  Hello good morning, I already finished this project alone last week I had to repeat the week because I wasn't able to complete my USP and my Quiz for the week, this happened because my coding partner was not present and he told me there was a protest going on in their country so we reschedule for another day, but he didn't show up I had no choice but to start instead of waiting for him which he ended up not showing up throughout that week it wasn't easy for me although you could check the previous review of my code it has already been approved thanks for your understanding.

**Comment by sja-thedude** [comment]
> @sja-thedude Hello good morning, I already finished this project alone last week I had to repeat the week because I wasn't able to complete my USP and my Quiz for the week, this happened because my coding partner was not present and he told me there was a protest going on in their country so we reschedule for another day, but he didn't show up I had no choice but to start instead of waiting for him which he ended up not showing up throughout that week it wasn't easy for me although you could check the previous review of my code it has already been approved thanks for your understanding.

I can understand but pair programming projects should be worked on with your coding partner. You worked solo last week because your partner had issues but this week you have another partner so you have to complete the project with the partner assigned this week.

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅
## Hi Team👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**



## Highlights:
* Well written PR description ✔️ 
* All tests passed ✔️ 
* The app is working properly and as expected ✔️ 


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**